packages/babel-plugin-transform-es3-reserved-words/test/index.js

```diff
@@ -0,0 +1,3 @@
+import runner from "babel-helper-plugin-test-runner";
```

Comment:
Also here

Reply:
Ahhh damn it, thanks!

Reply:
Haha now CI fails the other way around 😀

Reply:
Lol